services/credential-issuance-server/src/agent.ts

```diff
@@ -83,31 +123,39 @@ class Agent {
       !notif.r
     ) {
       switch (notif.a.r) {
-        case NotificationRoute.ExnIpexOffer: {
-          const msg = await this.signifyApi.getExchangeMsg(notif.a.d!);
-          await this.signifyApi.agreeToAcdcFromOffer(
-            Agent.ISSUER_AID_NAME,
-            msg.exn.d,
-            msg.exn.i
-          );
-          break;
-        }
-        default:
-          break;
+      case NotificationRoute.ExnIpexOffer: {
+        const msg = await this.signifyApi.getExchangeMsg(notif.a.d!);
+        await this.signifyApi.agreeToAcdcFromOffer(
+          Agent.HOLDER_AID_NAME,
+          msg.exn.d,
+          msg.exn.i
+        );
+        break;
+      }
+      default:
+        break;
       }
+      await this.signifyApi.deleteNotification(notif.i);
     }
-    await this.signifyApi.deleteNotification(notif.i);
   }
   async initKeri(issuerName?: string) {
     this.onNotificationKeriStateChanged();
     const AIDIssuerName = issuerName ? issuerName : Agent.ISSUER_AID_NAME;
-    const existedIndentifier = await this.signifyApi
+    const AIDHolderName = Agent.HOLDER_AID_NAME;
+    // Issuer
+    const existedIdentifier = await this.signifyApiIssuer
       .getIdentifierByName(AIDIssuerName)
       .catch(() => null);
-    if (existedIndentifier) return existedIndentifier;
-    const identifier = await this.signifyApi.createIdentifier(AIDIssuerName);
-    this.keriRegistryRegk = await this.signifyApi.createRegistry(AIDIssuerName);
-    return identifier;
+    if (existedIdentifier) return existedIdentifier;
```

Comment:
This function is getting messy. Lets do this:

```
  async initKeri(): Promise<void> {
    this.onNotificationKeriStateChanged();
    await this.signifyApiIssuer.createIdentifier(Agent.ISSUER_AID_NAME).catch(e => console.warn);
    await this.signifyApiIssuer.createRegistry(Agent.ISSUER_AID_NAME).catch(e => console.warn);
    await this.signifyApi.createIdentifier(Agent.HOLDER_AID_NAME).catch(e => console.warn);
    await this.signifyAPi.createRegistry(Agent.HOLDER_AID_NAME).catch(e => console.warn);
  }
```

So we just try to create everything but catch and log if it fails.

Reply:
As I said, this should be `Promise<void>` and remove the return statements. The catch allows us to continue in case the identifier/registry already exists